src/app/SpectralCounts.cpp

```diff
@@ -327,14 +327,17 @@ void SpectralCounts::normalizePeptideScores() {
     FLOAT_T score = it->second;
     total += score;
   }
+  // Avoid div by zero
+  if (total < 0.0000001) {  // TODO: what if total is negative?
```

Comment:
In case total is between [1e-7, 1.0], will it be better to use total = max(total, 1.0)?